controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -24,6 +24,17 @@ using namespace net;
 /////////////////////////// Local Module Functions ///////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
+static const std::string string_from_int_array(void *arr, size_t arr_max_size)
```

Comment:
The name of this function implies it creates a string from an array of integers. So it should be 
```
static const std::string string_from_int_array(int arr[], size_t len);
```
Also - please add a doxygen comment explaining this function.